modules/delegates.js

```diff
@@ -761,8 +761,11 @@ shared.getForgedByAccount = function (req, cb) {
 			return setImmediate(cb, err[0].message);
 		}
 
-		if (req.body.start && req.body.end) {
+		if (req.body.start !== undefined || req.body.end !== undefined) {
 			modules.blocks.aggregateBlocksReward({generatorPublicKey: req.body.generatorPublicKey, start: req.body.start, end: req.body.end}, function (err, reward) {
+				if (err) {
+					return setImmediate(cb, err || 'Error calculating block rewards');
```

Comment:
you will never be able to see the 'Error calculating block rewards' here, because 'err' satisfies the condition line above